app/views/content_items/service_sign_in/_choose_sign_in.html.erb

```diff
@@ -1,12 +1,31 @@
 <%= render "components/back-link", href: @content_item.back_link %>
 
+<% if @error %>
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <%= render "components/error-summary", {
+        title: t('service_sign_in.error.title'),
+        items: [
+          {
+            text: t('service_sign_in.error.option'),
+            href: "#option-0"
```

Comment:
Does this link to `id_prefix: "option"`?

Reply:
yes

Reply:
Ah, ok. It's not obvious they go together.